src/coord/src/coord.rs

```diff
@@ -280,41 +326,35 @@ impl Coordinator {
         self.worker_txs.len()
     }
 
-    /// Assign a timestamp for a read.
-    fn get_read_ts(&mut self) -> Timestamp {
-        let ts = self.get_ts();
-        self.last_op_was_read = true;
-        self.read_lower_bound = ts;
-        ts
+    fn system_timeline(&mut self) -> &mut Timeline {
+        self.timelines
+            .entry(TimelineId::EpochMilliseconds)
+            .or_insert_with(|| Timeline::new(0))
```

Comment:
Should this be `Timeline::new((now)())` or something like that?

Reply:
I don't know. My thought is that this function should not have an opinion about timestamp values and leave initialization up to other things. I'm going to work on linearizability across restarts next, and I believe that work will produce something that has an opinion how to init the system timeline if this is the initial start of mz.